django/apps/registry.py

```diff
@@ -91,21 +91,19 @@ def populate(self, installed_apps=None):
                     app_config = AppConfig.create(entry)
                 if app_config.label in self.app_configs:
                     raise ImproperlyConfigured(
-                        "Application labels aren't unique, "
-                        "duplicates: %s" % app_config.label)
+                        "Application labels aren't unique, " "duplicates: %s" % app_config.label
```

Comment:
This is one of the minor things that annoys me with `black` - it doesn't automatically fix implicit string concatenation.